### ISSUE TITLE
Use new token request procedure in the Poller

### ISSUE THREAD

**Issue by dchristidis** [rucio/rucio#6407]
### Description

 1. All-or-nothing approach: adapt the existing implementation to use the new token request procedure.
 2. Request-specific approach: investigate how to record whether a request was submitted with an X.509 certificate or with a token, and use the same authentication method to query for its state.

**Comment by robbarnsley** [comment]
Is it possible to implement a switch in the interim while this decision is being made @dchristidis? I'm having to use a custom image now to hardcode `oidc_support` for the poller to `True`, whereas I didn't in 1.29.6 (our previous version). 

I'm trying to get to a point where i'm just using the vanilla images as I easily lose track of these customisations during version updates...

**Comment by dchristidis** [comment]
Yes, certainly. Option 2 would require too much effort and re-engineering that, I believe, is unwarranted. I’ll start working on option 1.
